src/Fakes/FakeModelsCommand.php

```diff
@@ -66,21 +62,23 @@ public function getPropertiesFromTable($model): void
             return;
         }
 
-        $this->model_classes[] = get_class($model);
+        $this->model_classes[] = $model::class;
 
         $columns = $this->schema->tables[$table_name]->columns;
 
         foreach ($columns as $column) {
             $column_name = $column->name;
 
             if (in_array($column_name, $model->getDates(), true)) {
-                $get_type = $set_type = '\Illuminate\Support\Carbon';
+                $get_type = \Illuminate\Support\Carbon::class;
+                $set_type = \Illuminate\Support\Carbon::class;
             } else {
                 switch ($column->type) {
                     case SchemaColumn::TYPE_STRING:
                     case SchemaColumn::TYPE_INT:
                     case SchemaColumn::TYPE_FLOAT:
-                        $get_type = $set_type = $column->type;
+                        $get_type = $column->type;
+                        $set_type = $column->type;
```

Comment:
I didn't find this change valuable for the project, it actually creates more points of failure

Reply:
I'll skip the responsable Rector rule for this